eternalcore/src/main/java/com/eternalcode/core/user/PrepareUserController.java

```diff
@@ -36,4 +37,14 @@ public void onQuit(PlayerQuitEvent event) {
 
         user.setClientSettings(ClientSettings.NONE);
     }
+
+    @EventHandler
+    public void onKick(PlayerKickEvent event) {
+        Player player = event.getPlayer();
```

Comment:
Jak używasz tego playera 1 raz to po co robić zmienną?

Reply:
Żeby zachować styl kodu